OWNERS_ALIASES

```diff
@@ -361,10 +376,6 @@ aliases:
     - smarterclayton
     - thockin
 
-  sig-storage-api-approvers:
-    - liggitt
-    - thockin
```

Comment:
please restore this, we use it for targeting subsets of approvers with experience in different areas

Reply:
Should I also use the alias in the APIs folders for approvers?

Reply:
I used the alias in pkg/apis/storage and staging/src/k8s.io/api/storage

Let me know if that's undesired.